src/openms/source/FORMAT/HANDLERS/MzMLHandler.cpp

```diff
@@ -3377,14 +3377,22 @@ namespace OpenMS
 
     bool MzMLHandler::validateCV_(const ControlledVocabulary::CVTerm& c, const String& path, const Internal::MzMLValidator& validator) const
     {
-      SemanticValidator::CVTerm sc;
+      // static map<<Path, c.id>, isValid> for storing, if a cvterm is valid in the given path
+      const auto it = cached_terms_.find(std::make_pair(path, c.id));
```

Comment:
can you write a bit of local documentation to make clearer why the cache is useful?